.eslintrc.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
     ],
     "comma-dangle": ["error", "always-multiline"],
     "no-unused-vars": "off",
+    "@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
@kaylarobertson3 I would apply the eslint rule but just ignore the one line that was failing, as the rule is there for good coding practices. (using `// eslint-disable-line` or `// eslint-disable-next-line`)

Reply:
I think it was failing in a few files and I couldn't solve it in one without more time. but since this is merged it could definitely be a new ticket if you think so